File: src/main/java/com/ddubson/models/Genre.java
package com.ddubson.models;

import lombok.Builder;
import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "genres")
@Data
@Builder
public class Genre {
    @Id
    @GeneratedValue
    private Long id;

    private String name;
}

File: src/main/kotlin/com/ddubson/models/Genre.kt
package com.ddubson.models

import java.io.Serializable
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "genres")
data class Genre(@Id @GeneratedValue val id: Long,
                 val name: String) : Serializable {
}